godoc-current.txt

```diff
@@ -131,6 +131,77 @@ var MarkdownDocTemplate = `{{if gt .SectionNum 0}}% {{ .App.Name }} {{ .SectionN
 # COMMANDS
 {{ range $v := .Commands }}
 {{ $v }}{{ end }}{{ end }}`
+var MarkdownTabularDocTemplate = `{{ define "flags" }}
+| Name | Description | Default value | Environment variables |
+|------|-------------|:-------------:|:---------------------:|
+{{   range $flag := . -}}
```

Comment:
Do you need the $flag here or its just for convenience ?

Reply:
Yes, it's needed here, as it won't work without it :D